config/routes.rb

```diff
@@ -34,6 +34,7 @@
   get "/orders/new", to: "orders#new"
   post "/orders", to: "orders#create"
   get "/orders/:id", to: "orders#show"
+  patch "/orders/:order_id", to: "item_orders#update"
```

Comment:
I think a patch to "orders/:order_id" should be sending us to the orders controller.  I'm sorry, but could we have a conversation about this via slack?